### PR TITLE
Update to CodeNarc 1.1

### PR DESCRIPTION
**Pull request by nre-ableton**
This PR will update the CodeNarc JAR to version 1.1. Unfortunately the new indentation checks don't completely work as expected and must be disabled for now. ptal @AbletonDevTools/gotham-city, thanks!

### CODE REVIEW

File: ruleset.groovy [hunk 1]
```diff
@@ -29,7 +34,11 @@ ruleset {
     VariableName(ignoreVariableNames: jenkinsImportLibraryName)
   }
   ruleset('rulesets/security.xml') {}
-  ruleset('rulesets/size.xml') {}
+  ruleset('rulesets/size.xml') {
+    // This rule causes all sorts of CodeNarc crashes since version 1.1. We should
+    // consider re-enabling it in the future when those are ironed out.
+    AbcMetric(enabled: false)
```
Comment (ala-ableton) [inline ruleset.groovy]: @nre-ableton what is that rule about? Could you add an example?

**Comment by ala-ableton** [comment]
Only one question, then LGTM 👍

File: ruleset.groovy (hunk 1 above)
Comment (nre-ableton) [reply ruleset.groovy]: @ala-ableton http://codenarc.sourceforge.net/codenarc-rules-size.html#AbcMetric

I've never actually seen a violation of this in our code, but from what I understand it is checking to make sure that classes don't have too many methods. There is a much more complicated explanation of the ABC metric calculation here: https://en.wikipedia.org/wiki/ABC_Software_Metric

Comment (ala-ableton) [reply ruleset.groovy]: I didn't realise it was about "crashes", not "violations". Thanks for the explanation!

**Pull request merged**